paths_local.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2016 Mender Software AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+// +build local
+
+package main
+
+import (
+	"os"
+	"path"
+	"path/filepath"
+)
+
+func getRunningBinaryPath() string {
+	return filepath.Dir(os.Args[0])
+}
+
+func getDataDirPath() string {
+	return path.Join(getRunningBinaryPath(), "support")
+}
+
+func getStateDirPath() string {
+	return getRunningBinaryPath()
+}
+
+func getConfDirPath() string {
+	return getRunningBinaryPath()
+}
+
+func getBinDirPath() string {
+	return path.Join(getRunningBinaryPath(), "support")
```

Comment:
Right, I see the problem... Maybe, now that we've introduced `/usr/share/mender`, that it would be appropriate to put the identity script somewhere in there as well. After all, the identity script is never interesting as a standalone command, only when executed by Mender, hence it makes sense to put in a place local to Mender. What do you think?